console/module/account/src/main/java/org/eclipse/kapua/app/console/module/account/server/GwtAccountServiceImpl.java

```diff
@@ -203,27 +203,28 @@ public GwtAccount find(String accountIdString)
     }
 
     @Override
-    public ListLoadResult<GwtGroupedNVPair> getAccountInfo(String scopeIdString, String accountIdString)
+    public ListLoadResult<GwtGroupedNVPair> getAccountInfo(String scopeIdString, String accountIdString, String selectedAccountIdString)
             throws GwtKapuaException {
         final KapuaId scopeId = KapuaEid.parseCompactId(scopeIdString);
-        KapuaId accountId = KapuaEid.parseCompactId(accountIdString);
+        final KapuaId accountId = KapuaEid.parseCompactId(accountIdString);
+        final KapuaId selectedAccountId = KapuaEid.parseCompactId(selectedAccountIdString);
 
         List<GwtGroupedNVPair> accountPropertiesPairs = new ArrayList<GwtGroupedNVPair>();
         try {
-            final Account account = ACCOUNT_SERVICE.find(scopeId, accountId);
+            final Account account = ACCOUNT_SERVICE.find(scopeId, selectedAccountId);
 
             User userCreatedBy = KapuaSecurityUtils.doPrivileged(new Callable<User>() {
 
                 @Override
                 public User call() throws Exception {
-                    return USER_SERVICE.find(scopeId, account.getCreatedBy());
+                    return USER_SERVICE.find(accountId, account.getCreatedBy());
```

Comment:
This still does not work if the currently logger user's account is a parent of the currently selected account, does it?

Reply:
Leonardo and I are working on this. We have another solution, I will create PR tomorrow.

Reply:
@Coduz Yes, this solution still doesn't cover the case Leonardo mentioned - if the user from the child account makes a change the Created by/ Modified user information is not visible to the parent account.